jerry-core/parser/js/byte-code.h

```diff
@@ -576,6 +576,10 @@
               VM_OC_INITIALIZER_PUSH_PROP | VM_OC_GET_STACK) \
   CBC_FORWARD_BRANCH (CBC_EXT_DEFAULT_INITIALIZER, -1, \
                       VM_OC_DEFAULT_INITIALIZER) \
+  CBC_OPCODE (CBC_EXT_ERROR, CBC_NO_FLAG, 0, \
+              VM_OC_ERROR) \
+  CBC_FORWARD_BRANCH (CBC_EXT_BRANCH_IF_NULLISH, -1, \
+                      VM_OC_BRANCH_IF_NOT_UNDEFINED_NOR_NULL) \
```

Comment:
The corresponding VM instruction name should be adjusted as well.

Reply:
True i forgot the change it